static_test_driver/static_test_driver.ino

```diff
@@ -119,18 +125,19 @@ void setup()
     temp_status = true;
   }
   #endif
-  //-------------------Set up pressure sensors--------------------
+  //-------------------Set up pressure sensor--------------------
   pinMode (PRESSURE_PIN,INPUT);
+  //-------------------Set up force sensor-------------------
+  scale.set_scale(calibration_factor); //This value is obtained by using the SparkFun_HX711_Calibration sketch
+  scale.tare();  //Assuming there is no weight on the scale at start up, reset the scale to 0
```

Comment:
Does that mean I should tare after or before the statement READ_FLAG(start) { start_countdown(); } ?